teuthology/suite/util.py

```diff
@@ -201,6 +201,7 @@ def git_validate_sha1(project, sha1, project_owner='ceph'):
     we have specific urls to HEAD for github and git.ceph.com/gitweb
     for now
     '''
+    project_owner=config.project_owner
```

Comment:
This make `project_owner` argument useless.

Instead, if we really want to add this change, it might look like:
```
def git_validate_sha1(project, sha1, project_owner=None):
    url = build_git_url(project, project_owner or config.project_owner)
```